RUFAS/biophysical/manure/separators/separator.py

```diff
@@ -0,0 +1,277 @@
+        """
+        Takes in manure to be processed.
+
+        Parameters
+        ----------
+        manure : ManureStream
+            The manure to be processed.
+
+        """
+        if self.held_manure is None:
+            self.held_manure = manure
+        else:
+            self.held_manure += manure
+
+    def process_manure(self, conditions: CurrentDayConditions, time: Time) -> dict[str, ManureStream]:
+        """
+        Executes the daily manure processing operations.
+        For this class, it separates a portion of solids from the manure and returns the solid and liquid portions.
+
+        Parameters
+        ----------
+        conditions : CurrentDayConditions
+            Current weather and environmental conditions that manure is being processed in.
+        time : Time
+            Time instance containing the simulations temporal information.
+
+        Returns
+        -------
+        dict[str, ManureStream]
+            A dictionary containing:
+            - "solid" : ManureStream
+                The solid portion of the separated manure.
+            - "liquid" : ManureStream
+                The liquid portion of the separated manure.
+
+        """
+        info_map = {
+            "class": self.__class__.__name__,
+            "function": self.process_manure.__name__,
+        }
+        if not self.held_manure:
+            self.om.add_variable("empty_separator_output", {}, {**info_map, "units": MeasurementUnits.UNITLESS})
+            return {}
+        solid_manure_total_solids = self.held_manure.total_solids * self.total_solids_efficiency
+        solid_manure_total_mass = solid_manure_total_solids / self.percent_dry_solids
+        solid_manure_water = solid_manure_total_mass - solid_manure_total_solids
+        solid_manure_volume = (solid_manure_water + solid_manure_total_solids) / ManureConstants.SOLID_MANURE_DENSITY
+        solid_manure = ManureStream(
+            water=solid_manure_water,
+            ammoniacal_nitrogen=self.held_manure.ammoniacal_nitrogen * self.ammoniacal_nitrogen_efficiency,
+            nitrogen=self.held_manure.nitrogen * self.nitrogen_efficiency,
+            phosphorus=self.held_manure.phosphorus * self.phosphorus_efficiency,
+            potassium=self.held_manure.potassium * self.potassium_efficiency,
+            ash=self.held_manure.ash * self.ash_efficiency,
+            non_degradable_volatile_solids=self.held_manure.non_degradable_volatile_solids
+            * self.volatile_solids_efficiency,
+            degradable_volatile_solids=self.held_manure.degradable_volatile_solids * self.volatile_solids_efficiency,
+            total_solids=solid_manure_total_solids,
+            volume=solid_manure_volume,
+            pen_manure_data=None,
+        )
+        solid_stream_name = f"{self._prefix}.SeparatedSolids.{self._name}"
+        self.om.add_variable(
+            f"{solid_stream_name}.water", solid_manure.water, {**info_map, "units": MeasurementUnits.KILOGRAMS}
```

Comment:
```suggestion
            f"{solid_stream_name}.manure_water", solid_manure.water, {**info_map, "units": MeasurementUnits.KILOGRAMS}
```
I believe all of these variable names need to be prefixed with `manure_`

Reply:
Yes, if being reported to output manager, they should be prefixed with "manure_"